.vscode/settings.json

```diff
@@ -9,7 +9,4 @@
     "go.testEnvFile": "${workspaceFolder}/shared_files/decrypted/local.env",
     "go.testTimeout": "2m",
     "go.buildOnSave": "package",
-    "go.toolsEnvVars": {
```

Comment:
:heart:

Reply:
vscode users just have to restart the IDE for these changes to take affect, right?

Reply:
I believe so!